samples/bluetooth/mesh/light_ctrl/README.rst

```diff
@@ -205,13 +227,27 @@ The default value of :kconfig:option:`CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_PROLONG`
 .. note::
    The configuration of light levels, fade time, and timeouts can be changed by altering the configuration parameters in the :file:`prj.conf` file, and rebuilding the sample.
 
+Configure the Sensor Server model on the **Mesh Light Fixture** node:
+
+* Bind the model to **Application Key 1**, make sure to bind the same application key to the peer Sensor Client.
+
+The Sensor Server model is now configured and able to receive messages from and send data to the peer Sensor Client.
+
+Configure the Sensor Setup Server model on the **Mesh Sensor** node:
+
+* Bind the model to **Application Key 1**. make sure to bind the same application key to the peer Sensor Client.
+
```

Comment:
Should it say something about Publish/Subscriptions before it is able to receive sensor settings and it is configured? Same for sensor server.